src/components/custom/_signup-new.js

```diff
@@ -181,13 +191,12 @@ const SignupNew = ({
                         fontSize="var(--text-size-xs)"
                         translate_text="I agree to the <0>Terms and Conditions</0>"
                         components={[
-                            <StyledLink
-                                size="var(--text-size-xs)"
-                                weight="bold"
-                                key={0}
+                            <ExternalLink
+                                href="/terms-and-conditions/"
```

Comment:
since it's an external link to ourselves we need to consider the language for the route here also: `${lang}/terms-and-conditions/`